src/pipeline-component-factory.js

```diff
@@ -72,7 +73,12 @@ module.exports = function( nodeDef, wrapper, ondata ) {
  * 
  * @param node facade to be initialized
  */
-var pipelineNodeInit = function( wrapper, facade ) { 
-    vniManager( facade );
+var pipelineNodeInit = function(wrapper, facade) { 
+    vniManager(facade);
     facade.wrapper = wrapper || {};
+
+    // Set up the component profiler 
+    profiler(facade);
+    facade.profiler.createLog(__dirname+'/../logs',
```

Comment:
When packaged as a npm module, I don't think we should expect to be able to write to the node_modules directory.